layouts/shortcodes/details.html

```diff
@@ -1,4 +1,4 @@
 <details>
-  <summary><h2>{{ $.Params.summary }}</h2></summary>
+  <summary><h3>{{ $.Params.summary }}</h3></summary>
```

Comment:
Please move out all code changes into a new PR and have this as content only. Thanks!